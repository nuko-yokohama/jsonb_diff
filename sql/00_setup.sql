CREATE EXTENSION jsonb_diff;
\dx jsonb_diff;

