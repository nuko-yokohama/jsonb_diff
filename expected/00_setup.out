CREATE EXTENSION jsonb_diff;
\dx jsonb_diff;
                              List of installed extensions
    Name    | Version | Schema |                       Description                       
------------+---------+--------+---------------------------------------------------------
 jsonb_diff | 1.0     | public | Compare two JSONB documents and report the differences.
(1 row)

